### PR TITLE
feat: Shifted CI from Travis to Github Actions

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-2878

Pinned celery to v4.4.7 instead of v3.1.18 as v3.1.18 raised error used anyjson package which no longer exists and we're now using v4.4.7 in edx-platform.

### CODE REVIEW

File: requirements/test-master.txt [hunk 1]
```diff
@@ -35,34 +33,47 @@ botocore==1.18.18
     #   awscli
     #   boto3
     #   s3transfer
-celery==3.1.18
+celery==5.1.2
```
Comment (awais786) [inline requirements/test-master.txt]: Big Jump

**Review by mraarif — APPROVED** [review]
we should get approval from the owning team too

**Comment by Jawayria** [comment]
@edx/enterprise-titans please review

**Comment by Jawayria** [comment]
@macdiesel @adamstankiewicz please review

**Comment by macdiesel** [comment]
@muhammad-ammar What do you think about this. I believe you to be one of the persons who are using and modifying this repo the most. Could I ask you to take a look and merge this?

File: pylintrc_backup [hunk 2]
```diff
@@ -0,0 +1,458 @@
+# ***************************
```
Comment (muhammad-ammar) [inline pylintrc_backup]: can you please help understand why we have both the `pylintrc` and `pylintrc_backup`? Don't we need only one?

**Review by muhammad-ammar — APPROVED** [review]
LGTM.

File: pylintrc_backup (hunk 2 above)
Comment (Jawayria) [reply pylintrc_backup]: Yeah you're right. I'll remove it. It got added while running the command to update pylintrc.

**Pull request merged**